python/ray/autoscaler/updater.py

```diff
@@ -242,7 +251,7 @@ def do_update(self):
                             cli_logger.print(
                                 cmd_to_print, _numbered=("()", i, total))
 
-                            self.cmd_runner.run(cmd)
+                            self.cmd_runner.run(cmd, run_env="docker")
```

Comment:
is this supposed to be here? why is the run env == "docker" by default?

Reply:
Yes--it makes it clear that they are run inside docker for the DockerCommandRunner. For other CommandRunners, that is ignored.

Reply:
Hm, this can be confusing for others reading the code without docker context. Can this just be autodetected via the cmd_runner?

Something with the spirit of:
```
def DockerCommandRunner.run(cmd, run_env=None):
    if run_env is None: run_env = "docker"

Reply:
Sure, I'll add a comment